contribs/gnogenesis/internal/verify/verify.go

```diff
@@ -60,10 +63,32 @@ func execVerify(cfg *verifyCfg, io commands.IO) error {
 		}
 
 		// Validate the initial transactions
-		for _, tx := range state.Txs {
+		for index, tx := range state.Txs {
 			if validateErr := tx.Tx.ValidateBasic(); validateErr != nil {
 				return fmt.Errorf("invalid transacton, %w", validateErr)
 			}
+
+			// Genesis txs can only be signed by 1 account.
+			// Basic tx validation ensures there is at least 1 signer
+			signer := tx.Tx.GetSignatures()[0]
```

Comment:
Should we not check the length for empty signatures there? Or is it guaranteed to have at least one signature?